nixos/tests/dynomite.nix

```diff
@@ -0,0 +1,49 @@
+import ./make-test.nix ({ pkgs, lib, ... }: {
+  name = "dynomite";
+  meta.maintainers = [ lib.maintainers.Scriptkiddi ];
+
+  machine =
+    { ... }:
+    {
+      virtualisation.memorySize = 1024;
+      environment.systemPackages = with pkgs; [ redis dynomite ];
+      services.redis.enable = true;
```

Comment:
This shouldn't be needed with the automatic local redis detection

Reply:
done